core/domain/user_services.py

```diff
@@ -775,19 +775,72 @@ def get_user_contribution_rights(user_id):
         UserContributionRights. The UserContributionRights domain object for the
         corresponding user.
     """
-    user_model = (
-        user_models.UserContributionRightsModel.get_by_id(user_id))
-    if user_model is not None:
-        return user_domain.UserContributionRights(
-            user_id,
-            user_model.can_review_translation_for_language_codes,
-            user_model.can_review_voiceover_for_language_codes,
-            user_model.can_review_questions)
-    else:
-        return user_domain.UserContributionRights(user_id, [], [], False)
+    return get_users_contribution_rights([user_id])[0]
+
+
+def get_users_contribution_rights(user_ids):
+    """Returns the UserContributionRights domain object for each user_id in
+    user_ids.
+
+    Args:
+        user_ids: list(str). A list of user ids.
+
+    Returns:
+        list(UserContributionRights). A list containing the
+        UserContributionRights domain object for each user.
+    """
+    user_contribution_rights_models = (
+        user_models.UserContributionRightsModel.get_multi(user_ids)
+    )
+
+    users_contribution_rights = []
+    for index, user_contribution_rights_model in enumerate(
+            user_contribution_rights_models):
+        if user_contribution_rights_model is not None:
+            users_contribution_rights.append(user_domain.UserContributionRights(
+                user_ids[index],
+                (
+                    user_contribution_rights_model
+                    .can_review_translation_for_language_codes
+                ),
+                (
+                    user_contribution_rights_model
+                    .can_review_voiceover_for_language_codes
+                ),
+                user_contribution_rights_model.can_review_questions))
```

Comment:
Optional, but I suggest moving the model->domain object conversion to a helper method, possibly also handling the ``None`` case.

Reply:
Wouldn't it be more or less the same as the previous version? I was thinking something like:

```python
def _create_contribution_rights_from_model(user_id, user_contribution_rights_model):
    if user_contribution_rights_model is not None:
        return user_domain.UserContributionRights(
            ...)
    else:
        return user_domain.UserContributionRights(user_id, [], [], False)
```

It seems like a trivial thing to move to a helper, but I think it will result in two simpler functions that are a bit easier to follow. Or, am I missing something?

Reply:
I made a helper method but didn't provide the user_id as a param, just set the user_id to be an empty string if the model was None. In the `get_users_contribution_rights` method, if the model is None, I set the field to be the user_id. By doing it this way, I'm returning an empty UserContributionRights object if the model is None.

Reply:
I think that rationale makes sense--thanks!